tests/kernel/fp_sharing/testcase.yaml

```diff
@@ -12,3 +12,10 @@ tests:
     slow: true
     tags: kernel
     timeout: 600
+  kernel.fp_sharing.arc:
+    extra_args: PI_NUM_ITERATIONS=500
```

Comment:
How did you decide on this number of iterations?

Reply:
I choose this value dependes on the execution time of the test,  not too fast, not too slow.